app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@android:color/white"/>
+    <background android:drawable="@mipmap/ic_launcher_background"/>
```

Comment:
Thank you for taking this on @fieryprofessor, and welcome to the repo! :) 

(Not specifically related to this line, just adding a comment here so the conversation can remain threaded) @fieryprofessor It looks like this alters the launcher image and uses different colours than the ones defined in colors.xml. I'd prefer to stick to the simple launcher image in use (and the range of colours defined in colors.xml) and only make them compatible with monochromatic behaviour.

Additionally, the source of these changes is unclear - were these images and XML files created manually? was a third-party web service used to generate them? If so, what is the license of the output of the third-party web service?

Reply:
i actually downloaded the background and the icon from a website which was copright free, cant remember the exact details of it ,but anyways it was fun contributing to this project.